backend/clickhouse/migrations/000077_create_logs_replacing_merge_tree.down.sql

```diff
@@ -0,0 +1,3 @@
+EXCHANGE TABLES logs
+AND logs_new;
+DROP TABLE IF EXISTS logs_new;
```

Comment:
would going the other way technically require `attach partition` commands here? maybe we can have a comment to remember this for the future

Reply:
the partitions stay attached to the original table too - essentially the pointer to the part gets copied over to the new table but stays on the old one as well